docs/tutorials/platform_beginner/6_enabling_account_registration.md

```diff
@@ -0,0 +1,71 @@
+```markdown
```

Comment:
Not necessary, probably an artifact from some editor?

Reply:
Yep (I am currently trying Typora and it's quite satisfying except these kinds of traces)